### DISCUSSION TITLE
Mark Inbox as Read

### DISCUSSION THREAD

**Discussion by Tealk** [thunder-app/thunder#113] [Q&A] [answered]
Hi,

I haven't figured out yet how to mark posts that are in the inbox as read.
So both individually and all.

**Comment by hjiangsu** [comment, marked as answer]
Marking mentions and replies (individually) in the inbox as read will be out in the next release!

**Comment by pyonten** [comment]
Hey Guys, i already like your app so much, even in this 0.x state, amazing. I personally just need 1 more feature to make it my default lemmy app: mark posts as read by scrolling through the feed. hope this gets implemented some day, keep up the good work!!
